package.json

```diff
@@ -7,7 +7,7 @@
     "scripts": {
         "foreach-with-fallback": "yarn workspaces foreach --verbose --topological exec node ${PROJECT_CWD}/scripts/yarn-run-with-fallback.js $0 $1",
         "postinstall": "yarn run build",
-        "start-dev": "yarn workspaces foreach --verbose --parallel --interlaced run start-dev",
+        "start-dev": "yarn workspaces foreach --verbose --parallel --jobs unlimited --interlaced exec node ${PROJECT_CWD}/scripts/yarn-run-with-fallback.js start-dev global:start-dev",
```

Comment:
By default, yarn limits the number of parallel jobs, which is a problem in dev mode since the processes never exit.

I'm not sure if this "run `tsc --watch` for each package in parallel" approach is great... I suspect Typescript has a better way to help manage this. Something to look into, but not right now, I think.